src/main/resources/application.yml

```diff
@@ -29,7 +29,7 @@ server:
   port: 8443
 
 operations:
-  url: "http://bb-operations.mifos.io/api/v1"
+  url: https://ops-bk.sandbox.fynarfin.io/api/v1"
```

Comment:
These should be updated from the helm chart. not from the application Yml

Reply:
I have sent another PR in env template so you can close this PR